src/Common/tests/XunitTraitsDiscoverers/PlatformID.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    public enum PlatformID
```

Comment:
Am using a params array in the xunit attribute, so any number of platforms can be passed. I can also make this flags, which is preferred?